src/main/java/com/esl/demo/dto/ProductsOutletsDto.java

```diff
@@ -5,7 +5,9 @@
 
 public class ProductsOutletsDto {
 
-    private LinkId productsOutletsId;
+    private Long productId;
```

Comment:
add annotation validation on each field.

Reply:
That is a separate task.